cli/integration-test/entrypoint.clj

```diff
@@ -31,6 +32,29 @@
       (future-cancel fut))
     ret))
 
+(defn log-tail []
+  (:out (sh/sh "tail" "-n" "100" "/tmp/clojure-lsp.integration.out")))
```

Comment:
I already saw exceptions longer than 100, especially graalvm ones, WDYT about increasing it?
```suggestion
  (:out (sh/sh "tail" "-n" "300" "/tmp/clojure-lsp.integration.out")))
```

Reply:
Sure, MORE LOGS!